pom.xml

```diff
@@ -139,6 +139,16 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.8.0</version>
+          <configuration>
+            <release>${java.version}</release>
+          </configuration>
+          <dependencies>
+            <dependency>
+              <groupId>org.ow2.asm</groupId>
```

Comment:
@spulci do we need this? why?

Reply:
Removed...was a cut&paste error on my side sorry.